crm_expected_closing_kanban/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "CRM Expected Closing",
+    "summary": """
+        Added date_deadline field in the kanban view
```

Comment:
It is correct but it is preferable to add it in the present, not in the past, it would be like "Adds date_deadline field in kanban view"